### PR TITLE
remove irc page and point to gitlab instead

### PR DESCRIPTION
**Pull request by evykassirer**
addresses #248 
added some stuff from our page to the gitlab page so no info is lost: https://git.uwaterloo.ca/wics/documentation/wikis/irc

### CODE REVIEW

**Comment by annalorimer** [comment]
Accessing the GitLab isn't available to people who don't have UWaterloo credentials :(

**Comment by annalorimer** [comment]
Can people who don't have UWaterloo credentials still access the gitlab? I believe that people also have to be added to the GitLab project to see the wiki.

**Comment by evykassirer** [comment]
oh damn you do have to sign in

you don't need to be added to the project though, since the documentation project is public

I'll just copy the file over from the wiki to the irc page I guess?

**Comment by annalorimer** [comment]
Oh we copied the internal wiki to the documentation page? Nice. 

It's pretty lengthy to just copy and paste. Maybe make a separate page and link to it.

**Comment by evykassirer** [comment]
we didn't copy the internal wiki (I think, not sure what you mean)

there's multiple projects on gitlab, and the irc documentation is in the public project (the one wics-sys uses)

I'd copy and paste it to the separate page that already existed (that I just deleted)

**Comment by annalorimer** [comment]
I was refering to the wics-eyes-only wiki vs the wics/documentatin wiki. 

Sounds good!

**Comment by evykassirer** [comment]
oh I don't think it was ever on the internal wiki (or if it was, someone must've moved it over ^_^)

**Comment by annalorimer** [comment]
Oh you're right, it links to the documentation project. My bad!

File: content/pages/irc.md [hunk 1]
```diff
@@ -29,42 +29,81 @@ systems committee channel,
...
+on a screen.
+
+Steps:
+
+1. ssh into taurine or mother gooose, by running one of these commands in PuTTY
+   (Windows) or in a Terminal (Mac/Linux):
+   - `ssh userid@taurine.csclub.uwaterloo.ca` (for CSC) - If you don’t have an
+      account, visit the CSC office and pay $2 to sign up
+   - `ssh userid@mother-goose.wics.uwaterloo.ca` (for WiCS) - If you don’t have
+     a userid, ask Fatema to create one.
```
Comment (arshiamufti) [inline content/pages/irc.md:45]: I would link to the wics-sys mailing list instead of Fatema--this should fall under whoever is currently sysadmin

File: content/pages/irc.md [hunk 2]
```diff
@@ -29,42 +29,81 @@ systems committee channel,
...
-
-The instructions above are aimed at Computer Science Club members, who have
-login credentials for `taurine.csclub.uwaterloo.ca`. Unfortunately, at this
-time, WiCS does not offer community logins. We hope to have a server set up in
-the near future to provide this service. In the meantime, University of
-Waterloo students, staff, and alumni can purchase a
-[Computer Science Club](https://csclub.uwaterloo.ca) membership for $2 per
-term, which grants you access to the CSC network.
+#### WeeChat
+Weechat has become a favourite among committee, so if you like to have a gui
```
Comment (arshiamufti) [inline content/pages/irc.md:33]: among **the** committee?

File: content/pages/irc.md [hunk 3]
```diff
@@ -29,42 +29,81 @@ systems committee channel,
...
+6. To set up on your phone, download WeeChat Android, or the Glowing Bear app,
+   and go to settings>connection settings, fill out the same information again
+7. Either in the app or the web interface, you need to join all the channels
+   and servers. In the buffer type:
+    - `/server add oftc irc.oftc.net`
+    - `/connect oftc`
+8. Go to the oftc screen, in glowing-bear click on the bear’s face, and click
+   on oftc, in weechat android click on the W in the top-left corner
+    - `/join #wics` (public conversation)
+    - `/join #wics-sys`
```
Comment (arshiamufti) [inline content/pages/irc.md:63]: I would add a description of the purpose of `wics-sys`

File: content/pages/irc.md [hunk 4]
```diff
@@ -29,42 +29,81 @@ systems committee channel,
...
-the near future to provide this service. In the meantime, University of
-Waterloo students, staff, and alumni can purchase a
-[Computer Science Club](https://csclub.uwaterloo.ca) membership for $2 per
-term, which grants you access to the CSC network.
+#### WeeChat
+Weechat has become a favourite among committee, so if you like to have a gui
+and point-and-clickness, we recommend you try this first. It also has
+notifications on phone and desktop, and you are always connected if you keep it
+on a screen.
+
```
Comment (arshiamufti) [inline content/pages/irc.md:37]: I think we can move the bit that says "you need an account on taurine or mother-goose before setting up IRC" and also instructions on how to do that (go to CSC with $2/ email wics-sys) before the steps.

File: content/pages/irc.md [hunk 5]
```diff
@@ -29,42 +29,81 @@ systems committee channel,
...
+- `/nick new_name` (e.g. `/nick evy`): If you want to change your nickname use
+  this command  where `new_name` is the name you want. If that nick is
+  registered, you have 30 seconds to "identify" yourself
+- `/msg nickserv identify $password`: this is the command used to identify
+  yourself. Your password is whatever you set it to be when you register your
+  nick
+- `/join $channel` (e.g. `/join #wics`): This is the command used to join a
+  channel
+- `/q $nick` (e.g. `/q evy`): This opens a "query" window with another user.
+  This is used to PM someone. You can then switch to that window and chat with
```
Comment (arshiamufti) [inline content/pages/irc.md:91]: "PM" -> "PM (private message or direct message)" ?

File: content/pages/irc.md (hunk 1 above)
Comment (evykassirer) [reply content/pages/irc.md:45]: yeah, definitely. good point

File: content/pages/irc.md (hunk 5 above)
Comment (evykassirer) [reply content/pages/irc.md:91]: I think I'll just get rid of PM and say 'privately message'

**Comment by evykassirer** [comment]
Thanks for the feedback Arshia - I just copied and pasted from our wiki and this should really be clearer than that doc ^_^

File: content/pages/irc.md [hunk 6]
```diff
@@ -18,9 +18,9 @@ we do not publish our logs. Communication in the channel is governed by our
 [Code of Conduct]({filename}/pages/code-of-conduct.md). If you haven't given it
 a read yet, please take a few minutes to familiarize yourself with it!
 
-# Installation instructions #
+## Setting Up IRC (you can pick any of these options)
```
Comment (annalorimer) [inline content/pages/irc.md:21]: I think "Use whichever option works best for you" would be better on the line below the title. It's an instruction, so it doesn't super fit being in the header.

File: content/pages/irc.md (hunk 2 above)
Comment (annalorimer) [reply content/pages/irc.md:33]: I would shorten this to "WeeChat is good if you like to have a GUI". I find instructions really hard to follow when they're wordy.

File: content/pages/irc.md [hunk 7]
```diff
@@ -29,42 +29,81 @@ systems committee channel,
...
+     a userid, ask Fatema to create one.
+2. type into the terminal `screen weechat`
+3. in WeeChat buffer type: `/set relay.network.password [pick a password that
+   you don’t use for anything important]`
+4. `/relay add weechat #####` where ##### is a port number. Pick a number
+   between 28000 and 28500
+5. To set up on a web browser, go to glowing-bear.org, and enter the
+   information, (eg. host: taurine.uwaterloo.ca; port: 00009; password:
+   butts123)
+6. To set up on your phone, download WeeChat Android, or the Glowing Bear app,
```
Comment (annalorimer) [inline content/pages/irc.md:54]: Having 5 & 6 as separate steps makes it seem like you have to do both to get WeeChat to work. Maybe combine this in to a single step?

File: content/pages/irc.md [hunk 8]
```diff
@@ -29,42 +29,81 @@ systems committee channel,
...
+   and go to settings>connection settings, fill out the same information again
+7. Either in the app or the web interface, you need to join all the channels
```
Comment (annalorimer) [inline content/pages/irc.md:56]: In this step, it says that "you need to join all channels" but that doesn't actually happen in this step. This can probably be combined with 8.

File: content/pages/irc.md [hunk 9]
```diff
@@ -29,42 +29,81 @@ systems committee channel,
...
+
+Steps:
+
+1. ssh into taurine or mother gooose, by running one of these commands in PuTTY
+   (Windows) or in a Terminal (Mac/Linux):
+   - `ssh userid@taurine.csclub.uwaterloo.ca` (for CSC) - If you don’t have an
+      account, visit the CSC office and pay $2 to sign up
+   - `ssh userid@mother-goose.wics.uwaterloo.ca` (for WiCS) - If you don’t have
+     a userid, ask Fatema to create one.
+2. type into the terminal `screen weechat`
```
Comment (annalorimer) [inline content/pages/irc.md:46]: The word after the numbers should be capitalized :)

**Comment by annalorimer** [comment]
@evykassirer @arshiamufti What's the status of this?

**Comment by evykassirer** [comment]
haven't been working on it for a while since it's not urgent. I'll try to get it done this term though, or someone else can take it on if they want!

**Comment by annalorimer** [comment]
It'd be nice if we could sort this out soon... It's been open for a while and I don't think it'll take long to fix.

**Comment by evykassirer** [comment]
well, I don't think it's more important than #249 which took months

If I do it, it'll probably be in the next month. I'm also happy for someone else to take it on if we want it sooner :)

**Comment by annalorimer** [comment]
@wics-uw/contributors It'd be nice to get feedback from people currently trying to use the documentation :) Let us know what you think needs fixing.
